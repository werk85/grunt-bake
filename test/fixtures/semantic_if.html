<!DOCTYPE html>
<html>
	<head></head>
	<body>
		<!--(bake includes/project.html
    		usesPHP="no"
    		usesNodeJS="yes"
    		usesASP="off")-->
	</body>
</html>