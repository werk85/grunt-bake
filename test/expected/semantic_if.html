<!DOCTYPE html>
<html>
	<head></head>
	<body>
		<dt>

			<dt>NodeJS</dt>
			<dd>This project uses NodeJS.</dd>

		</dt>
	</body>
</html>